src/ontology/metadata/ncit-metrics.json

```diff
@@ -4,8 +4,8 @@
     "abox_axiom_count_incl": 0,
     "annotation_property_count": 12,
     "annotation_property_count_incl": 12,
-    "axiom_count": 623839,
-    "axiom_count_incl": 623839,
+    "axiom_count": 1328208,
```

Comment:
## FYI: NCIT axioms increase
Note that this has to do with NCIT and not us, but this is a huge increase in axioms.
It looks like most of them are `IAO` and `oboInOwl`

Also I was thinking this kind of thing might be better on Slack instead of PR, but there is no way for me to link a diff into Slack without adding a comment, it looks like, so I'll just leave the comment here instead.

Reply:
If this seems important enough to add as a FYI comment in the PR then add it. However, a simple one-liner will suffice.

Reply:
Here is good - it is a significant observation. I assume this is because of our receny harmonisation of NCIT synonym properties.